homeassistant/components/anthemav/__init__.py

```diff
@@ -1 +1,59 @@
-"""The anthemav component."""
+"""The Anthem A/V Receivers integration."""
+from __future__ import annotations
+
+import logging
+
+import anthemav
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .const import ANTHEMAV_UDATE_SIGNAL, DOMAIN
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Anthem A/V Receivers from a config entry."""
+
+    @callback
+    def async_anthemav_update_callback(message):
+        """Receive notification from transport that new data exists."""
+        _LOGGER.debug("Received update callback from AVR: %s", message)
+        async_dispatcher_send(hass, f"{ANTHEMAV_UDATE_SIGNAL}_{entry.data[CONF_NAME]}")
```

Comment:
We shouldn't use a user configured string as an id for the dispatch signal. That's fragile. We can use the config entry entry_id instead.